board/qualcomm/qcom-sdm660-phone.config

```diff
@@ -0,0 +1,61 @@
+# Phone-friendly environment
+CONFIG_ENV_DEFAULT_ENV_TEXT_FILE="board/qualcomm/qcom-sdm660-phone.env"
+
+# Support booting Android boot images
+CONFIG_BOOTMETH_ANDROID=y
+CONFIG_CMD_ABOOTIMG=y
+
+# "Nice to have"
+CONFIG_CMD_POWEROFF=y
+CONFIG_CMD_UFETCH=y
+CONFIG_CMD_SELECT_FONT=y
+CONFIG_CMD_PMIC=y
+CONFIG_VIDEO_FONT_4X6=y
+CONFIG_VIDEO_FONT_8X16=y
+CONFIG_VIDEO_FONT_SUN12X22=y
+# SDM630/636/660 don't have UFS, they need eMMC/SD card support
+CONFIG_SD_BOOT=y
+CONFIG_QCOM_UFS=n
+CONFIG_CLK_QCOM_SDM660=y
+CONFIG_PINCTRL_QCOM_SDM660=y
+# HS200 works perfectly fine here
+CONFIG_MMC_HS200_SUPPORT=y
+# Some devices, e.g. Sony logo is black on white background.
+# Very annoying. Clear framebuffer.
+CONFIG_NO_FB_CLEAR=n
+
+# We don't need PSHOLD, SYSRESET_PSCI is working fine for us
+CONFIG_SYSRESET_CMD_POWEROFF=n
+CONFIG_SYSRESET_QCOM_PSHOLD=n
+
+# Hang on panic so the error message can be read
```

Comment:
This section and below is copied from upstream's `qcom-phone.config`. Perhaps I'll mention it